pliers/transformers/microsoft.py

```diff
@@ -0,0 +1,80 @@
+import os
+import requests
+from pliers.transformers import Transformer
+from pliers.utils import EnvironmentKeyMixin
+
+
+BASE_URL = 'https://{location}.api.cognitive.microsoft.com/{api}/{version}'\
+           '/{method}'
+
+
+class MicrosoftAPITransformer(Transformer):
+    ''' Base MicrosoftAPITransformer class.
+
+    Args:
+        subscription_key (str): a valid subscription key for Microsoft Cognitive
+            Services. Only needs to be passed the first time the extractor is
+            initialized.
+        location (str): region the subscription key has been registered in.
```

Comment:
I think other users might have the same confusion I did, so maybe add a bit more explanation about where this is found (and give an example value or two). Also, does this have to match the user's region? If it all it does is help optimize response latencies, we could probably set a default. I imagine most users won't really care if it takes a few ms longer to get a response.